src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs

```diff
@@ -141,7 +145,12 @@ public async Task<IActionResult> OnGetAsync(string id, string revisionId = null)
             {
                 foreach (var username in approverConfig.Split(","))
                 {
-                    approvers.Add(username);
+                    UserProfileModel user = await _userProfileRepository.tryGetUserProfileByNameAsync(username);
+                    var langs = user.Languages;
+                    if (langs.Contains(Review.Language.ToLower()))
```

Comment:
This will be broken if language name is updated as per my suggestion on other PR.

Reply:
Removed to ToLower() call